run_forecast.py

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env python3
+
+import argparse
+import budgettool as bt
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser(description="generate csv file of transactions from budget")
```

Comment:
To do later:  It works well to have a function that constructs and returns the parser, leaving a single line/function call in main().